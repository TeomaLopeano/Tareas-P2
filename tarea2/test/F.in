agregarAIterador 31
agregarAIterador 32
reversoDeIterador
agregarAIterador 33
agregarAIterador 34
agregarAIterador 35
agregarAIterador 36
agregarAIterador 37
agregarAIterador 38
agregarAIterador 39
reversoDeIterador
agregarAIterador 40
agregarAIterador 41
agregarAIterador 42
agregarAIterador 43
agregarAIterador 44
reversoDeIterador
Fin