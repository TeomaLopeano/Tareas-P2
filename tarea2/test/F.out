1>Agregando 31 a it.
2>Agregando 32 a it.
3>32 31 
4>Agregando 33 a it.
5>Agregando 34 a it.
6>Agregando 35 a it.
7>Agregando 36 a it.
8>Agregando 37 a it.
9>Agregando 38 a it.
10>Agregando 39 a it.
11>39 38 37 36 35 34 33 32 31 
12>Agregando 40 a it.
13>Agregando 41 a it.
14>Agregando 42 a it.
15>Agregando 43 a it.
16>Agregando 44 a it.
17>44 43 42 41 40 39 38 37 36 35 34 33 32 31 
18>Fin.
